test/unit/definition.test.ts

```diff
@@ -46,21 +46,14 @@ describe('API definition class', () => {
         const api = await API.load('examples/valid/asyncapi.v2.yml');
         expect(api.version).to.equal('2.2.0');
         expect(api.references.length).to.equal(5);
-        expect(api.references.map((ref) => ref.location)).to.include(
-          'http://example.org/param-lights.json',
-        );
+        const locations = api.references.map((ref) => ref.location);
```

Comment:
Can we add a test for Windows path?

Reply:
I thought about it, but I remember I had troubles running tests on a windows worker. I'll try it again before merging